### PR TITLE
add github profile

### PR DESCRIPTION
**Pull request by gcsecsey**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -12,16 +12,18 @@ export default function App() {
...
         <a href="https://www.facebook.com/incremental.hu/">
-          <SVGIcon style={{ marginRight: ".5em" }} name="facebook" />
+          <SVGIcon style={{ padding: ".5em" }} name="facebook" />
+        </a>
+        <a href="https://github.com/Incremental-Consulting">
+          <SVGIcon style={{ padding: ".5em" }} name="github" />
         </a>
         <a href="https://www.linkedin.com/company/incremental-consulting">
-          <SVGIcon name="linkedin" />
+          <SVGIcon style={{ padding: ".5em" }} name="linkedin" />
```
Comment (Adam3306) [inline src/App.js:26]: Why did u do that

File: src/components/SVGIcon.js [hunk 2]
```diff
@@ -37,6 +37,15 @@ const getPath = (name, props) => {
           <path {...props} d="m91 180h90v-90h-90zm30-60h30v30h-30zm0 0" />
         </>
       );
+    case "github":
```
Comment (Adam3306) [inline src/components/SVGIcon.js:40]: "github" move to constants

**Review by Adam3306 — CHANGES_REQUESTED** [review]

File: src/components/SVGIcon.js [hunk 3]
```diff
@@ -1,6 +1,6 @@
 import React from "react";
 
-const getViewBox = name => {
+const getViewBox = (name) => {
   switch (name) {
     case "facebook":
```
Comment (Adam3306) [inline src/components/SVGIcon.js:5]: move to constants

File: src/components/SVGIcon.js [hunk 4]
```diff
@@ -1,6 +1,6 @@
 import React from "react";
 
-const getViewBox = name => {
+const getViewBox = (name) => {
```
Comment (Adam3306) [inline src/components/SVGIcon.js:3]: is it just a formatting change? if yes revert it

File: src/App.js (hunk 1 above)
Comment (gcsecsey) [reply src/App.js:26]: Will move this to a flexbox container in the next PR. :)

File: src/components/SVGIcon.js (hunk 2 above)
Comment (gcsecsey) [reply src/components/SVGIcon.js:40]: I'm planning on introducing constants once there are a high number of icons.

File: src/components/SVGIcon.js (hunk 4 above)
Comment (gcsecsey) [reply src/components/SVGIcon.js:3]: I'd like to keep formatting consistent

**Pull request merged**